build/Build.cs

```diff
@@ -119,52 +119,18 @@ private static string GetOTelAutoInstrumentationFileName()
         .After(this.Compile)
         .Executes(() =>
         {
-            // TODO: fix build script to copy dependencies without manually setting them here.
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "Newtonsoft.Json.dll",
-                this.openTelemetryDistributionFolder / "net");
-
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "OpenTelemetry.Extensions.AWS.dll",
-                this.openTelemetryDistributionFolder / "net");
-
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "OpenTelemetry.ResourceDetectors.AWS.dll",
-                this.openTelemetryDistributionFolder / "net");
-
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "OpenTelemetry.SemanticConventions.dll",
-                this.openTelemetryDistributionFolder / "net");
-
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "AWS.Distro.OpenTelemetry.AutoInstrumentation.dll",
-                this.openTelemetryDistributionFolder / "net");
-
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "OpenTelemetry.Instrumentation.AWS.dll",
-                this.openTelemetryDistributionFolder / "net");
-
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "OpenTelemetry.Sampler.AWS.dll",
-                this.openTelemetryDistributionFolder / "net");
-
-            FileSystemTasks.CopyFileToDirectory(
-                RootDirectory / "src" / "AWS.Distro.OpenTelemetry.AutoInstrumentation" / "bin" / this.configuration /
-                "net8.0" / "AWSSDK.Core.dll",
-                this.openTelemetryDistributionFolder / "net");
+            FileSystemTasks.CopyDirectoryRecursively(
```

Comment:
Do we need to copy all files from net6.0 folders?

Reply:
Because it’s easier honestly, the size difference is around 800kb. Previously, everytime we add a new dependency we had to worry about making sure we copy the file to the correct location etc but now we don't.